spec/examples/running_cluster.rb

```diff
@@ -76,9 +76,7 @@ def get_postgres_extensions
   end
 
   def run_agent(agent_name)
-    # setting up a Windows TTY is difficult, so we don't
-    # allocating a TTY will show container pull output on Linux, but that's not good for tests
-    result = run_command("docker run --rm --network pupperware_default --name #{agent_name} --hostname #{agent_name} puppet/puppet-agent-alpine")
+    result = run_command("docker run --rm --network pupperware_default --dns #{@consul_ip} --name #{agent_name} --hostname #{agent_name} puppet/puppet-agent-alpine agent -t --server puppet.service.consul --ca_server puppet")
```

Comment:
I think we still want to keep the comments I just added?

Reply:
oh sorry bad rebase